lib/services/elasticsearch.js

```diff
@@ -73,6 +75,13 @@ class ElasticSearch extends Service {
       return Bluebird.resolve(this);
     }
 
+    if (process.env.NODE_ENV === 'production' && this.config.dynamic === 'true') {
+      this.kuzzle.pluginsManager.trigger('log:warn',
+        'Your dynamic mapping policy is set to \'true\' for new fields.\n\
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use string concatenation rather than line continuation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AMultilineStringLiterals)